test_autofit/graphical/info/test_info.py

```diff
@@ -37,23 +37,23 @@ def test_non_trivial_info(
     assert info == """PriorFactors
 
 PriorFactor0 (AnalysisFactor1.sigma)                                                      UniformPrior, lower_limit = 0.0, upper_limit = 1.0
-PriorFactor1 (AnalysisFactor1.intensity)                                                  UniformPrior, lower_limit = 0.0, upper_limit = 1.0
+PriorFactor1 (AnalysisFactor1.normalization)                                              UniformPrior, lower_limit = 0.0, upper_limit = 1.0
```

Comment:
These are the kinds of changes that affect identifiers

Reply:
This model component is only used for testing / documentation so its fine, but indeed I wouldnt be doing changes like this in autolens without a lot of care.